examples/bridge-app/bridge-common/gen/call-command-handler.cpp

```diff
@@ -338,10 +338,61 @@ EmberAfStatus emberAfOnOffClusterServerCommandParse(EmberAfClusterCommand * cmd)
             wasHandled = emberAfOnOffClusterOffCallback();
             break;
         }
+        case ZCL_OFF_WITH_EFFECT_COMMAND_ID: {
```

Comment:
Why these changes here?

Reply:
Those changes are not intentional. 

This is yet an other side effect of the submodule update. I will turn them back off.